Makefile

```diff
@@ -58,3 +58,7 @@ k8s: validate
 .PHONY: 1.18
 1.18:
 	$(MAKE) k8s kubernetes_version=1.18.9 kubernetes_build_date=2020-11-02 pull_cni_from_github=true
+
+.PHONY: 1.19
+	1.19:
```

Comment:
I think the indentation is off here for a Makefile

Reply:
Ya, you're right. I'll fix that.